package.json

```diff
@@ -5,14 +5,16 @@
   "main": "lib/index.js",
   "typings": "./index.d.ts",
   "scripts": {
+    "qa": "run-p lint test",
     "lint-src": "./node_modules/.bin/eslint src/**/*.js",
     "lint-test": "./node_modules/.bin/eslint test/**/*.js",
-    "lint": "npm run lint-src && npm run lint-test",
+    "lint": "run-p lint-*",
     "prepublish": "./node_modules/.bin/babel src --out-dir lib",
     "pretest": "./node_modules/.bin/babel src --out-dir lib",
-    "test-run": "./node_modules/.bin/mocha --require babel-core/register",
-    "test": "npm run lint && npm run test-run",
-    "prepare": "npm run prepublish"
+    "test": "mocha --require babel-core/register",
+    "prepare": "npm run prepublish",
+    "precommit": "lint-staged",
+    "prepush": "yarn qa"
```

Comment:
I'm not familiar enough with yarn - is this a yarn only script?

Reply:
This sure looks like a typo. I would say this is supposed to be `npm run qa`